plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -206,6 +202,34 @@ func NewServiceAntiAffinityPriority(podLister algorithm.PodLister, serviceLister
 	return antiAffinity.CalculateAntiAffinityPriority
 }
 
+// Classifies nodes into ones with labels and without labels.
+func(s *ServiceAntiAffinity) getNodeClassificationByLabels(nodes []*v1.Node) (map[string]string, []string){
```

Comment:
Sure, thats the main idea but I wanted to check, if there is something I missed.